Sources_v3/Controllers/ChannelController_Tests.swift

```diff
@@ -499,6 +662,37 @@ class ChannelController_Tests: StressTestCase {
         AssertAsync.willBeEqual(completionCalledError as? TestError, testError)
     }
 
+    func test_showChannel_blocked_by_isChannelCreatedFlag() {
+        //  Create `ChannelController` for new channel
+        let query = ChannelQuery(channelPayload: .unique)
+        setupControllerForNewChannel(query: query)
+
+        var channelError: Error?
+
+        // Simulate `showChanne;` call
+        controller.showChannel() { [callbackQueueID] error in
+            AssertTestQueue(withId: callbackQueueID)
+            channelError = error
+        }
+
+        // Assert error is returned if method called before channel creation
+        AssertAsync.willBeTrue(channelError is ClientError.ChannelDoesNotExist)
+
+        // Simulate succsesfull backend channel creation
+        env.channelUpdater!.update_channelCreatedCallback?(query.cid)
+
+        var nilError: Error?
+
+        // Simulate `showChannel` call
+        controller.showChannel() { [callbackQueueID] error in
+            AssertTestQueue(withId: callbackQueueID)
+            nilError = error
+        }
```

Comment:
The callback of this function is never called, so the `nilError` will always be `nil`. I found it thanks to refactoring using `await`.

Reply:
Forgot to mention that I fixed it there by calling `env.channelUpdater!.showChannel_completion?(nil)`